pixels-core/src/main/java/io/pixelsdb/pixels/core/PixelsWriterImpl.java

```diff
@@ -519,6 +520,14 @@ private void writeRowGroup()
                     byte[] rowGroupBuffer = writer.getColumnChunkContent();
                     physicalWriter.append(rowGroupBuffer, 0, rowGroupBuffer.length);
                     writtenBytes += rowGroupBuffer.length;
+                    int alignment = Integer.parseInt(ConfigFactory.Instance().getProperty("row.group.align"));
```

Comment:
Rename it to chunkAlignment and make a private static final member of the class.